src/test/java/org/spectrumauctions/sats/opt/examples/CustomizedExamples.java

```diff
@@ -1,6 +1,7 @@
 package org.spectrumauctions.sats.opt.examples;
 
 import com.google.common.collect.ImmutableMap;
+import org.junit.Ignore;
```

Comment:
Since you agree on the main changes, I'm going to merge this for the new release, i hope that's ok. We can still change the ignored examples again in the future.